scipy/ndimage/tests/test_filters.py

```diff
@@ -740,6 +748,31 @@ def test_filter_axes(self, filter_func, extra_args, size0, size, axes):
         expected = filter_func(array, *extra_args, all_sizes)
         assert_allclose(output, expected)
 
+    @pytest.mark.parametrize('func', [ndimage.correlate, ndimage.convolve])
+    @pytest.mark.parametrize('dtype', [np.float32, np.complex64])
```

Comment:
Is there a reason not to test `np.float64` and `np.complex128`? Double precision is uncommon for image data but you never know..

Reply:
I was just trying not to introduce too many total combinations since data types weren't the main point of the axes tests, but went ahead and added it. 

The full set of ndimage tests only take 6-7 seconds when I ran it locally.